packages/react/src/molecules/pickers/select/base.tsx

```diff
@@ -48,6 +48,7 @@ export interface PickerSelectBaseProps {
   closeComponent?: any;
   clearComponent?: any;
   noDataComponent?: any;
+  customRightComponent?: any;
```

Comment:
Pls Use a better type.For example if it is a component then JSX.Element or React.ReactNode(if it can be string , number or other datatypes)